eo-runtime/src/main/eo/org/eolang/collections/list.eo

```diff
@@ -95,7 +95,25 @@
         &.f x > @
 
   # For each array element dataize the object
-  [f] > each /bool
+  # Here "f" must be an abstract object with
+  # one free attribute, the element of the
+  # array.
+  [f] > each
+    [f] > iterate
+      seq > @
+        memory 0 > idx
+        idx.write 0
+        while.
+          lt.
+            idx
+            arr.length
+          [i]
+            seq > @
+              f (arr.at i)
+              idx.write (i.plus 1)
+    eq. > @
```

Comment:
@l3r8yJ According to [this](https://arxiv.org/pdf/2206.02585.pdf) and [this](https://262.ecma-international.org/5.1/#sec-15.4.4.18) - there is no need to mutate the object that called the `list.each` object. Therefore, I propose to replace it with `TRUE`.

Reply:
@Graur when i call `ˆ.as-bytes.eq 0` and it's turn into 'TRUE`, can we say that object empty or null?

Reply:
@l3r8yJ please, read [this](https://arxiv.org/abs/2111.13384): `9 Key Features:` `No NULL References` and `No Empty Objects`

Reply:
@Graur thanks!